app/web/components/CenteredSpinner/index.ts

```diff
@@ -0,0 +1 @@
+export { default } from "./CenteredSpinner";
```

Comment:
I think you can get rid of this file, we don't really follow the index.js pattern for exports here. It's fine enough to just `export default` in the component itself.

Though making one file where we export all our custom components from is certainly something we could discuss to decrease the length of all our file imports! Maybe not a decision for this ticket though.

Reply:
Okay good to know! will remove